server/services/postgres/alembic/versions/2020-04-20T102810_add_flow_concurrency_limits.py

```diff
@@ -0,0 +1,55 @@
+"""
+add flow concurrency limits
+
+Revision ID: 52f08a976090
+Revises: 7e65dadba625
+Create Date: 2020-04-20 10:28:10.913791
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects.postgresql import JSONB, UUID
+
+# revision identifiers, used by Alembic.
+revision = "52f08a976090"
+down_revision = "7e65dadba625"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute(
+        """
+        -- Table Definition ----------------------------------------------
+
+        CREATE TABLE flow_concurrency_limit (
+            id uuid DEFAULT gen_random_uuid() PRIMARY KEY,
+            created timestamp with time zone NOT NULL DEFAULT now(),
+            updated timestamp with time zone NOT NULL DEFAULT now(),
+            name character varying NOT NULL UNIQUE,
+            description text,
```

Comment:
There's a description column here, but I don't think the ability to modify it is exposed to the user. I'm inclined to say we should nix it for the sake of parity with Cloud, but I'm interested to hear your thoughts.

Reply:
Correct, it isn't able to modified currently. The end goal was to allow users to put arbitrary text explaining either why the limit is in place, what sets of infrastructure is limited by it (E.g. EKS cluster 1, EKS cluster 2), or anything similar. The end idea was to allow users to edit this in the UI, and I'm personally still a fan of surfacing this kind of functionality, but definitely okay with removing it from the table in the short term in favor of having a separate PR that adds specifically that functionality!